README.md

```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
-                Setpoint item=LandroidScheduleSaturdayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleSaturdayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleSaturdayEdgecut
-            }
-            Frame label="Schedule Sunday" {
-                Setpoint item=LandroidScheduleSundayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleSundayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleSundayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleSundayEdgecut
-            }
+	Group item=Shaun icon="landroid" {
+        Frame {
+			Switch item=LandroidAction label="Action" mappings=[START="Start"] visibility=[LandroidStatusCode==0, LandroidStatusCode==1]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==7]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==33]
+			Switch item=LandroidAction label="Action" mappings=[START="Start",HOME="Home"] visibility=[LandroidStatusCode==34]
+			Switch item=LandroidPoll label="Refresh" mappings=[ON="Poll"]
+            Text item=LandroidLastUpdate
+            Switch item=LandroidLock label="Lock" mappings=[ON="LOCK",OFF="UNLOCK"]
         }
-        Text label="MultiZone" icon="pie"{
-            Frame label="Zone Meters" {
-                Slider item=LandroidMeterZone1 minValue=0 maxValue=150
-                Slider item=LandroidMeterZone2 minValue=0 maxValue=150
-                Slider item=LandroidMeterZone3 minValue=0 maxValue=150
-                Slider item=LandroidMeterZone4 minValue=0 maxValue=150
-            }
-            Frame label="Allocation Zones" {
-                Switch item=LandroidAllocation0 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation1 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation2 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation3 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation4 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation5 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation6 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation7 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation8 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation9 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
+        Frame {
+            Text item=LandroidMacAdress
```

Comment:
please remove LandroidMacAdress

Reply:
?? why, is this not right place to add this?

Reply:
removed => solved